prime-router/docs/design/design/RS-Extention-Registry.yml

```diff
@@ -0,0 +1,3773 @@
+                    "valueCodeableConcept": {
+                      # CWE to CodeableConcept
+                    }
+                  },
+                  {
+                    "url": "PV1.18",
+                    "valueCodeableConcept": {
+                      # CWE to CodeableConcept
+                    }
+                  },
+                  {
+                    "url": "PV1.46",
+                    "valueDecimal": 55
+                  },
+                  {
+                    "url": "PV1.47",
+                    "valueDecimal": 1000
+                  },
+                  {
+                    "url": "PV1.48",
+                    "valueDecimal": 0
+                  },
+                  {
+                    "url": "PV1.49",
+                    "valueDecimal": 0
+                  },
+                  {
+                    "url": "PV1.51",
+                    "valueCodeableConcept": {
+                      # CWE to CodeableConcept
+                    }
+                  },
+                  {
+                    "url": "PV1.30",
+                    "valueString": "T"
+                  },
+                  {
+                    "url": "PV1.31",
+                    "valueCodeableConcept": {
+                      # CWE to CodeableConcept
+                    }
+                  },
+                  {
+                    "url": "PV1.32",
+                    "valueDecimal": 3.4
+                  },
+                  {
+                    "url": "PV1.33",
+                    "valueDecimal": 1.4
+                  },
+                  {
+                    "url": "PV1.34",
+                    "valueCodeableConcept": {
+                      # CWE to CodeableConcept
+                    }
+                  },
+                  {
+                    "url": "PV1.35",
+                    "valueString": "20240501102531-0400"
+                  },
+                  {
+                    "url": "PV1.39",
+                    "valueCodeableConcept": {
+                      # CWE to CodeableConcept
+                    }
+                  },
+                  {
+                    "url": "PV1.41",
+                    "valueCodeableConcept": {
+                      # CWE to CodeableConcept
+                    }
+                  }
+                ],
+                "class": {
+                  "extension": [
+                    {
+                      "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/pv1-2-patient-class",
+                      "valueCodeableConcept": {
+                        # CWE to CodeableConcept
+                      }
+                    }
+                  ],
+                  "system": "http://terminology.hl7.org/CodeSystem/v3-ActCode",
+                  "code": "AMB",
+                  "display": "ambulatory"
+                }
+              }
+            ]
+          }
+        }
+      Extension:
+        - id: hl7v2Field
```

Comment:
This doesn't have a description, which could make sense, but I'm wondering if we want to provide maybe a section at the top that explains these generic extensions?